pkg/gofr/datasource/redis/redis.go

```diff
@@ -27,11 +33,13 @@ func NewRedisClient(config RedisConfig) (*redis.Client, error) {
 	}
 
 	rc := redis.NewClient(config.Options)
+	rc.AddHook(&redisHook{logger: logger})
+
 	if err := rc.Ping(context.TODO()).Err(); err != nil {
```

Comment:
context should be with timeout duration

Reply:
Sure will add a context with a timeout of `5 sec`